lib/utils.js

```diff
@@ -263,12 +263,15 @@ exports.configureHook = function (hook, defaults, root) {
 
 // Configure a default script by name and content
 // For example, Validate.installScript('test', 'lab -a code -L');
-exports.installScript = function (name, script, root) {
+// Or Validate.installScript('test', 'lab -a code -L', {overwrite: true}); to force updating
+exports.installScript = function (name, script, options, root) {
 
+    options = options || {};
```

Comment:
let's add some code here to check if `typeof options === 'string'` then we `root = options; options = {}`, then you can revert your changes to the existing tests and only add new tests